doc/mgr/ansible.rst

```diff
@@ -41,25 +41,46 @@ Enable the Ansible orchestrator module and use it with the :ref:`CLI <orchestrat
 Configuration
 =============
 
+The external Ansible Runner Service uses TLS mutual authentication to allow clients to use the API.
+A client certificate and a key files should be provided by the Administrator of the Ansible Runner Service for each manager node.
+This files should be copied in each of the manager nodes with read access for the ceph user.
+The destination folder for this files and the name of the files must be the same always in all the manager nodes,
+althought the certificate/key content of this files logically will be different in each node.
+
 Configuration must be set once the module is enabled by first time.
 
 This can be done in one monitor node via the configuration key facility on a
 cluster-wide level (so they apply to all manager instances) as follows::
 
+In first place, configure the Ansible Runner Service client certificate an key:
+
+    If the client certificate provided is for several servers (generic) use:
+    # ceph ansible set-ssl-certificate -i <location_of_the_crt_file>
+    # ceph ansible set-ssl-certificate-key -i <location_of_the_key_file>
 
-    # ceph config set mgr mgr/ansible/server_addr <ip_address/server_name>
-    # ceph config set mgr mgr/ansible/server_port <port>
-    # ceph config set mgr mgr/ansible/username <username>
-    # ceph config set mgr mgr/ansible/password <password>
-    # ceph config set mgr mgr/ansible/verify_server <verify_server_value>
+    If the client certificate provided is for an especific manager server use:
+    # ceph ansible set-ssl-certificate <server> -i <location_of_the_crt_file>
+    # ceph ansible set-ssl-certificate-key <server> -i <location_of_the_key_file>
+
+After setting the client certificate & key files to use, finish configuration as follows:
```

Comment:
After setting the client certificate and key files, finish the configuration as follows:

Reply:
Thx!. Done